### ISSUE TITLE
Update to Search bar

### ISSUE THREAD

**Issue by SchoolofCommons** [cubeca/cube#409]
Unable to search all of school of commons content on the main search page. 
User example: User types 'School of Commons' and see's everything we have uploaded on the main search page or at least the icon to navigate to our profile page.

**Comment by AshBC** [comment]
## <a href="https://gov4git.org"><img src="https://raw.githubusercontent.com/gov4git/gov4git/main/materials/gov4git-avatar.png" alt="This project is governed with Gov4Git." width="65" /></a> Gov4Git notices
On `Tuesday, 01-Oct-24 20:28:11 UTC` by Gov4Git `dev`

#### Notice `34vjhx`
Started managing this issue, using the Open-Source Management Protocol (Waimea), as Gov4Git concern `409` with initial __priority score__ of `0.000000`.

This project is managed by [Gov4Git](https://gov4git.org), a decentralized governance system for collaborative git projects.
To participate in governance, __install the [Gov4Git desktop app](https://github.com/gov4git/desktop-application)__.
	

#### Notice `6uwy6t`
This issue's __priority score__ is now `0.000000`.
The __cost of priority__ is `0.000000`.
The __projected bounty__ is now `0.000000`.

#### Notice `wux3ld`
The set of eligible proposals claiming this issue is empty.

**Comment by AshBC** [comment]
## <a href="https://gov4git.org"><img src="https://raw.githubusercontent.com/gov4git/gov4git/main/materials/gov4git-avatar.png" alt="This project is governed with Gov4Git." width="65" /></a> Gov4Git notices
On `Tuesday, 01-Oct-24 23:11:18 UTC` by Gov4Git `dev`

#### Notice `cjcd25`
This issue's __priority score__ is now `5.000000`.
The __cost of priority__ is `25.000000`.
The __projected bounty__ is now `10.000000`.

#### Notice `y4jenq`
The set of eligible proposals claiming this issue is empty.

**Comment by TheRealArtengine** [comment]
Yes I think the developers response makes sense and is more scalable long term.

**Comment by AshBC** [comment]
@taylorjackson estimated 4 hours for this.

**Comment by AshBC** [comment]
## <a href="https://gov4git.org"><img src="https://raw.githubusercontent.com/gov4git/gov4git/main/materials/gov4git-avatar.png" alt="This project is governed with Gov4Git." width="65" /></a> Gov4Git notices
On `Friday, 29-Nov-24 18:36:02 UTC` by Gov4Git `dev`

#### Notice `6lhbqa`
This issue's __priority score__ is now `4.000000`.
The __cost of priority__ is `8.000000`.
The __projected bounty__ is now `8.000000`.

#### Notice `ldug4a`
The set of eligible proposals claiming this issue is empty.

**Comment by AshBC** [comment]
Currently in the bottom 3 - might not occur in 2025 depending on dev hours available

**Comment by AshBC** [comment]
This issue what resolved with pull request 467
